### PR TITLE
fix: fix undefined prop key error

### LINKED ISSUE #8971: TypeError: Cannot read properties of undefined (reading 'trim')
Sentry Issue: [POSTHOG-59M](https://sentry.io/organizations/posthog2/issues/3086668080/?referrer=github_integration)

```
TypeError: Cannot read properties of undefined (reading 'trim')
  at Ak (../src/lib/components/PropertyKeyInfo.tsx:648:52)
...
(15 additional frame(s) were not displayed)
```

### PR DESCRIPTION
**Pull request by alexkim205**
## Problem

Fixes https://github.com/PostHog/posthog/issues/8971

### CODE REVIEW

File: frontend/src/lib/components/PropertyKeyInfo.tsx [hunk 1]
```diff
@@ -645,7 +645,7 @@ export function PropertyKeyInfo({
     value = `${value}` // convert to string
 
     const data = getKeyMapping(value, type)
-    const baseValue = (!data ? value : data.label).trim()
+    const baseValue = (!data ? value : data.label)?.trim() ?? ''
```
Comment (liyiy) [inline frontend/src/lib/components/PropertyKeyInfo.tsx:648]: nit but I think things are more readable if it doesn't have to be negated

```suggestion
    const baseValue = (data ? data.label : value)?.trim() ?? ''
```

**Review by liyiy — APPROVED** [review]

File: frontend/src/lib/components/PropertyKeyInfo.tsx (hunk 1 above)
Comment (alexkim205) [reply frontend/src/lib/components/PropertyKeyInfo.tsx]: good call 👍

**Comment by neilkakkar** [comment]
Why was this happening only for that one project? 🤔

**Comment by alexkim205** [comment]
> Why was this happening only for that one project? 🤔

Another user reported it so it seems more ubiquitous which makes sense - https://posthogusers.slack.com/archives/C01GLBKHKQT/p1646939628108819

**Comment by neilkakkar** [comment]
Asking because we may be patching over a more subtle bug (idk yet 🤷‍♂️, sorry for being annoying): 

Then the question is why was this not happening for my personal project + PostHog project?

**Comment by neilkakkar** [comment]
Hmm, since I'm support hero and all, I guess I can be less annoying by trying to answer this myself!

What's interesting is that there are properties like:

image

(Raw json string: `'__lookupGetter__'`)

which are now all rendered as `(empty string)`.

image

which seems wrong to me? These shouldn't be PostHog properties at all?

**Comment by alexkim205** [comment]
Hmm, I can't seem to replicate this issue. I've tried it with both collaborations_taxonomy on and off but its telling me that it's being processed correctly as non PH properties. Could you possibly send me a link where your're seeing this?


Screen Shot 2022-03-11 at 12 08 38 PM
Screen Shot 2022-03-11 at 12 08 07 PM

**Pull request merged**